custom_components/mikrotik_router/coordinator.py

```diff
@@ -517,10 +520,37 @@ def get_capabilities(self):
             self.support_wireless = True
             if "wifiwave2" in packages and packages["wifiwave2"]["enabled"]:
                 self.support_wifiwave2 = True
+                self.support_wifi = False
                 self.support_capsman = False
+                self._wifimodule = "wifiwave2"
+                
+            elif "wifi" in packages and packages["wifi"]["enabled"]:
+                self.support_wifiwave2 = False
```

Comment:
with this change, self.support_wifiwave2 and self.support_wifi are most likely unused variables.

Reply:
Okay, removing self.support_wifiwave2 and self.support_wifi